src/edu/kpi/ip71/dovhopoliuk/Main.java

```diff
@@ -1,19 +1,19 @@
 package edu.kpi.ip71.dovhopoliuk;
 
 import edu.kpi.ip71.dovhopoliuk.caesar.CaesarXorSolution;
+import edu.kpi.ip71.dovhopoliuk.polyalphabetic.PolyalphabeticSolution;
 import edu.kpi.ip71.dovhopoliuk.railfence.RailFenceSolution;
 import edu.kpi.ip71.dovhopoliuk.substitution.SubstitutionSolution;
 import edu.kpi.ip71.dovhopoliuk.vigenere.VigenereXorSolution;
 
-import java.util.List;
-
 public class Main {
 
     public static void main(final String... args) {
 
 //        new RailFenceSolution().solve("resources/railfence/input.txt", "resources/railfence/output.txt");
 //        new CaesarXorSolution().solve("resources/caesar/input.txt", "resources/caesar/output.txt");
 //        new VigenereXorSolution().solve("resources/vigenere/input.txt", "resources/vigenere/output.txt");
-        new SubstitutionSolution().solve("resources/substitution/input.txt", "resources/substitution/output.txt");
+//        new SubstitutionSolution().solve("resources/substitution/input.txt", "resources/substitution/output.txt");
+        new PolyalphabeticSolution().solve("resources/polyalphabetic/input.txt", "resources/polyalphabetic/output.txt");
```

Comment:
@CodeGangsta44 I would recommend uncommenting those lines as lab 01 is finished

Reply:
Thanks, no.